test/e2e/resources/notebook_instance_lifecycle_retain.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: NotebookInstanceLifecycleConfig
+metadata:
+  name: $NOTEBOOK_INSTANCE_LFC_NAME
+  annotations:
```

Comment:
not a big issue but it will be good to reuse existing spec since we are just adding an annotation

Reply:
done